tools/clang/lib/SPIRV/TypeTranslator.cpp

```diff
@@ -619,6 +619,16 @@ bool TypeTranslator::isMxNMatrix(QualType type, QualType *elemType,
   return true;
 }
 
+bool TypeTranslator::isRowMajorMatrix(QualType type, const Decl *decl) const {
```

Comment:
I'm not entierly sure which types to actually use here... Hopefully this is not too far off.

Reply:
LGTM.